### ISSUE TITLE
Kiwi uses outdated analyzer dependencies

### ISSUE THREAD

**Issue by vanlooverenkoen** [vanlooverenkoen/kiwi#21]
```Because no versions of kiwi_generator match >0.4.0 <0.5.0 and kiwi_generator 0.4.0 depends on analyzer >=0.32.4 <0.37.0, kiwi_generator ^0.4.0 requires analyzer >=0.32.4 <0.37.0.

And because json_serializable >=3.2.3 depends on analyzer >=0.37.1 <0.39.0, kiwi_generator ^0.4.0 is incompatible with json_serializable >=3.2.3.

So, because my_app depends on both json_serializable ^3.2.3 and kiwi_generator ^0.4.0, version solving failed.
pub get failed (1; So, because my_app depends on both json_serializable ^3.2.3 and kiwi_generator ^0.4.0, version solving failed.)
```

This should be fixed

**Comment by arnemolland** [comment]
Until this is updated, here's a temporary fix:
```yaml
dependency_overrides:
  analyzer: ^0.39.0
```

**Comment by vanlooverenkoen** [comment]
@letsar are you still planning to update your package?

**Comment by vanlooverenkoen** [comment]
Fix in kiwi 0.3.0 & kiwi_generator 0.5.0
